AUTHORS

```diff
@@ -0,0 +1,78 @@
+Alexander Rose <alexander.rose@weirdbyte.de>
```

Comment:
don't know how that got there. will remove.

thanks for the feedback. I'll make the requested changes shortly.